rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2021-11-07
+nightly
```

Comment:
this better to be a nightly version which is stable enough for debugging (without too much ICE)

Reply:
and `nightly-2021-11-07` is also used in CI, if we would like to change, we should pick another better version then replace in all places

Reply:
sorry, I just want to change something to commit and trigger CI, I will change this back now.